app/src/main/java/org/oppia/app/home/HomeFragmentController.kt

```diff
@@ -34,4 +50,19 @@ class HomeFragmentController @Inject constructor(
   private fun getUserAppHistoryViewModel(): UserAppHistoryViewModel {
     return viewModelProvider.getForFragment(fragment, UserAppHistoryViewModel::class.java)
   }
+
+  fun goToExplorationButtonClicked(v: View) {
```

Comment:
Here & below: maybe instead 'playExplorationButton' would be clearer. I was at first a bit confused because it seems like we're just routing to the exploration, but what we're really doing is starting to play an exploration.

Reply:
Updated to `playExplorationButton` in this file, xml file and test-case too.